### PR TITLE
Adding a section explaining OLE clients and servers behavior for Office version 2109 and newer.

### PR DESCRIPTION
**Pull request by JakubK44**
The section discusses how Excel and PowerPoint handle DPI awareness compatibility when acting as OLE clients/servers, which changed in version 2109.

### CODE REVIEW

**Comment by JakubK44** [comment]
I'll fix the absolute paths issue throughout the page.

File: docs/ddpi/handle-high-dpi-and-dpi-scaling-in-your-office-solution.md [hunk 1]
```diff
@@ -529,23 +529,39 @@ m_currentDPI = ::GetDpiForWindow(this->GetSafeHwnd());
...
+
+#### Before Office 365 version 2109
+
+If Word or Excel detect that there are multiple monitors with different DPI scales, they will not support in-place activation. Your OLE server will activate out-of-place. If you are experiencing issues with OLE server interactions, the user will need to use [Office DPI compatibility mode](https://support.office.com/article/office-support-for-high-definition-displays-6720ca0e-be59-41f6-b629-1369f549279d).
+
+#### After Office 365 version 2109
+
+Starting with version 2109 for Excel and PowerPoint, both applications allow in-place activation in Per Monitor DPI Aware modes when certain conditions are met. When in-place activated as OLE servers, Office applications will check the `DPI_AWARENESS_CONTEXT` of the parent window provided by the container and allow in-place activation if it's compatible with [When using multiple displays](https://support.microsoft.com/en-us/topic/office-support-for-high-definition-displays-6720ca0e-be59-41f6-b629-1369f549279) setting of the Office app:
```
Comment (davidchesnut) [inline docs/ddpi/handle-high-dpi-and-dpi-scaling-in-your-office-solution.md:549]: Looks like some text is jumbled or something at "if it's compatible with [When using multiple displays]"

File: docs/ddpi/handle-high-dpi-and-dpi-scaling-in-your-office-solution.md [hunk 2]
```diff
@@ -529,23 +529,39 @@ m_currentDPI = ::GetDpiForWindow(this->GetSafeHwnd());
...
+- "Optimize for best appearance" will allow in-place activation if the window provided by the container is Per Monitor DPI Aware
```
Comment (davidchesnut) [inline docs/ddpi/handle-high-dpi-and-dpi-scaling-in-your-office-solution.md:550]:
```suggestion
- "Optimize for best appearance" will allow in-place activation if the window provided by the container is Per Monitor DPI Aware.
```

File: docs/ddpi/handle-high-dpi-and-dpi-scaling-in-your-office-solution.md [hunk 3]
```diff
@@ -529,23 +529,39 @@ m_currentDPI = ::GetDpiForWindow(this->GetSafeHwnd());
...
+- "Optimize for compatibility" will allow in-place activation if the window provided by the container is System DPI aware and system DPI matches between both the container application and the Office application
```
Comment (davidchesnut) [inline docs/ddpi/handle-high-dpi-and-dpi-scaling-in-your-office-solution.md:551]:
```suggestion
- "Optimize for compatibility" will allow in-place activation if the window provided by the container is System DPI aware and both the container application and the Office application have the same system DPI.
```

File: docs/ddpi/handle-high-dpi-and-dpi-scaling-in-your-office-solution.md [hunk 4]
```diff
@@ -529,23 +529,39 @@ m_currentDPI = ::GetDpiForWindow(this->GetSafeHwnd());
...
+
+It means that DPI Unaware OLE containers will not be able to in-place activate either of the apps. The workaround is to change the DPI awareness of the container app to System DPI Aware (and use the "Optimize for compatibility" setting in the Office app). This can be done via [SetProcessDpiAwareness](/windows/win32/api/shellscalingapi/nf-shellscalingapi-setprocessdpiawareness), [embedded or external manifest](/windows/win32/hidpi/setting-the-default-dpi-awareness-for-a-process). External manifest is especially useful for existing VB Forms solutions, as it does not require the solution to be recompiled.
```
Comment (davidchesnut) [inline docs/ddpi/handle-high-dpi-and-dpi-scaling-in-your-office-solution.md:553]: What does "It" refer to in the first sentence?

File: docs/ddpi/handle-high-dpi-and-dpi-scaling-in-your-office-solution.md [hunk 5]
```diff
@@ -529,23 +529,39 @@ m_currentDPI = ::GetDpiForWindow(this->GetSafeHwnd());
...
+
+When acting as an OLE container, both Excel and PowerPoint defer the DPI awareness compatibility check to the server and do not prevent in-place activation.
```
Comment (davidchesnut) [inline docs/ddpi/handle-high-dpi-and-dpi-scaling-in-your-office-solution.md:555]:
```suggestion
When acting as an OLE container, both Excel and PowerPoint defer to the server to check DPI awareness compatibility, and they do not prevent in-place activation.
```

**Review by davidchesnut — CHANGES_REQUESTED** [review]
Looks good, a few comments and suggestions.

File: docs/ddpi/handle-high-dpi-and-dpi-scaling-in-your-office-solution.md [hunk 6]
```diff
@@ -529,23 +529,39 @@ m_currentDPI = ::GetDpiForWindow(this->GetSafeHwnd());
...
+
+#### Before Office 365 version 2109
+
+If Word or Excel detect that there are multiple monitors with different DPI scales, they will not support in-place activation. Your OLE server will activate out-of-place. If you are experiencing issues with OLE server interactions, the user will need to use [Office DPI compatibility mode](https://support.office.com/article/office-support-for-high-definition-displays-6720ca0e-be59-41f6-b629-1369f549279d).
+
+#### After Office 365 version 2109
+
+Starting with version 2109 for Excel and PowerPoint, both applications allow in-place activation in Per Monitor DPI Aware modes when certain conditions are met. When in-place activated as OLE servers, Office applications will check the `DPI_AWARENESS_CONTEXT` of the parent window provided by the container and allow in-place activation depending on which of the following [Office DPI display modes](https://support.microsoft.com/en-us/topic/office-support-for-high-definition-displays-6720ca0e-be59-41f6-b629-1369f549279d) is selected by the user:
```
Comment (davidchesnut) [inline docs/ddpi/handle-high-dpi-and-dpi-scaling-in-your-office-solution.md:549]:
```suggestion
Starting with version 2109 for Excel and PowerPoint, both applications allow in-place activation in Per Monitor DPI Aware modes when certain conditions are met. When in-place activated as OLE servers, Office applications will check the `DPI_AWARENESS_CONTEXT` of the parent window provided by the container and allow in-place activation depending on which of the following [Office DPI display modes](https://support.microsoft.com/topic/office-support-for-high-definition-displays-6720ca0e-be59-41f6-b629-1369f549279d) is selected by the user:
```

**Review by davidchesnut — APPROVED** [review]
Looks great!

**Pull request merged**